### DISCUSSION TITLE
Make it possible to link DLL directly

### DISCUSSION THREAD

**Discussion by ghost** [LADSoft/OrangeC#988] [Ideas]
Currently, I have to make an import library using `oimplib`.

**Comment by GitMensch** [comment]
It doesn't work if you pass it on the occ and/or olink command line? Then please make a FR issue out of that, this feature in GCC on Windows is quite useful and it would be good to have that with OCC.

**Reply by ghost** [reply]
See #992.

**Comment by LADSoft** [comment]
we did add this, so if it doesn't work I've broken something somewhere.    Will evaluate soon...

**Reply by GitMensch** [reply]
Any update?

**Comment by LADSoft** [comment]
this seems to work for me....   perhaps the linkage in your test isn't defined the same in the dll as it is in the source code using it...   or perhaps you are trying to use C++ functions (which wouldn't be supported if the DLL came from a different compiler).
